frontend/src/components/FundingContent.tsx

```diff
@@ -0,0 +1,56 @@
+import { nofoPdfs } from "src/constants/nofoPdfs";
+
+import { useTranslation } from "next-i18next";
+import { Grid, GridContainer } from "@trussworks/react-uswds";
+
+import PdfContainer from "./PdfContainer";
+
+const FundingContent = () => {
+  const { t } = useTranslation("common", { keyPrefix: "Index" });
+
+  return (
+    <div className="bg-base-lightest">
+      <GridContainer>
+        <Grid row>
+          <h2 className="margin-bottom-0">{t("fo_title")}</h2>
+        </Grid>
+        <Grid row gap="md">
+          <Grid tablet={{ col: 6 }}>
+            <p>{t("fo_paragraph_1")}</p>
+          </Grid>
+          <Grid tablet={{ col: 6 }}>
+            <p>{t("fo_paragraph_2")}</p>
+          </Grid>
+        </Grid>
+        <Grid>
+          <h3>{t("fo_title_2")}</h3>
+        </Grid>
+        <Grid>
+          <p className="usa-intro">{t("fo_paragraph_3")}</p>
```

Comment:
I noticed this in the Goal Content component, that the fonts are kinda wonky and we're swapping between serif and sans. Just an observation we may want to address now or in the future. 

The cause according to testing locally is the `usa-intro` class. I'm guessing in some cases is applies a different font palette that overrides our current one? Not entirely sure, should we remove?

Reply:
(In Figma, I failed to represent the `.usa-intro` content with the right serif font. Not a problem that this doesn't match what's in Figma.)